cairo-contracts/scripts/compile_all_cairo1.sh

```diff
@@ -0,0 +1,228 @@
+#!/bin/bash
```

Comment:
`cairo-contracts/scripts/compile_all_cairo1.sh`

I though it would compile both cairo1 and cairo0

Reply:
Yeah i forgot to change the name 🤡